CHANGELOG.md

```diff
@@ -6,6 +6,24 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project does not yet adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 for setuptools_scm/PEP 440 reasons.
 
+## 1.2.6 Chia blockchain 2021-09-09
+
+### Changed
+
+- Updated to BLS 1.0.6.
+- Updates to the Rust conditions parser.
+- Adjusted plot refresh parameter to improve plot loading times.
+
+### Fixed
+
+- Fixed memory leak in versions 1.2.4 and 1.2.5 related to how the node handles compact VDFs generated from blueboxes.
```

Comment:
All versions - not just 1.2.4/1.2.5

Reply:
Also missing

- Fixed issues with reloading plot files detected as bad (this can happen during plot copying)

Reply:
Thx will fix @emlowe